HelicsBool handling and comments modification

afisher1 opened the pull request:
first attempt at adding % in front of all function comments to make use of help feature in MATLAB.
also modified HelicsBool arguments to accept
logcials or int/float values of 0 and 1.

phlptp commented on matlabBindings/+helics/helicsBrokerSetLogFile.m:
```diff
@@ -1,10 +1,10 @@
+%	@param logFileName The name of the file to log to.
 
-	@param[in,out] err An error object that will contain an error code and string if any error occurred during the execution of the function.
+%	@param[in,out] err An error object that will contain an error code and string if any error occurred during the execution of the function.
```
Don't need to do this now but we might need to have some sort of script that goes through and cleans these up.  For example the err param is not used in matlab

phlptp commented:
@afisher1  is this change included in the 3.3 stuff you did or is this separate?

nightlark commented:
It looks like the changes here were not included in PR #10

afisher1 commented:
reading the MATLAB documentation the help text needs to be just below the function definition in the .m files so I'll need to change that.

afisher1 commented:
I made the requested changes to the function help comments. removed the multi comment brackets and removed all occurences of param[in,out] err lines from the generated functions.
